app/components/ScorecardProgress/ScorecardProgressShareButton.js

```diff
@@ -41,21 +29,12 @@ class ScorecardProgressShareButton extends Component {
     });
   }
 
-  renderWarningIcon() {
-    if (!this.props.hasMatchedEndpointUrl)
-      return <MaterialIcon name='error' size={17} color={ Color.whiteColor }
-                style={{ position: 'absolute', top: -6, right: -8}}
-             />
-  }
-
   render() {
     return (
-      <TouchableOpacity onPress={() => this.shareSubmittedScorecard()}
-        disabled={ !this.state.isShareable }
+      <TouchableOpacity onPress={() => this.shareSubmittedScorecard()} disabled={ !this.props.scorecard.isCompleted }
```

Comment:
@limkimsan please consider use disabled={!this.props.scorecard.isShareable()} instead of scorecard.isCompleted